package-scripts/datadog-agent/postrm

```diff
@@ -3,8 +3,15 @@ LINUX_DISTRIBUTION=$(grep -Eo "(Debian|Ubuntu|RedHat|CentOS|openSUSE|Amazon)" /e
 
 if [ -f /etc/debian_version ] || [ "$LINUX_DISTRIBUTION" == "Debian" ] || [ "$LINUX_DISTRIBUTION" == "Ubuntu" ]; then
     set -e
+    
+    # Removing the service form startup (since it's not there anymore)
+    if command -v systemd >/dev/null 2>&1; then
+        systemctl disable datadog-agent
```

Comment:
Is there an `update-rc.d disable` that would do the opposite of `update-rc.d enable` you used in the postinst script?